src/components/projects/ManuscriptPageContainer.tsx

```diff
@@ -62,10 +65,19 @@ const ManuscriptPageContainer: React.FC = () => {
 
 const ManuscriptPageView: React.FC = () => {
   const can = usePermissions()
+  const [isTrackingOn, setTrackingOn] = useState(true)
```

Comment:
Can you please move the functionality introduced by this PR into a custom hook and call it here if possible? This is because ManuscriptPageView is a relatively high level component and if we can abstract away an implementation of a feature, that could help keep it cleaner. What do you think?

Reply:
Done